pumps/mongo_aggregate.go

```diff
@@ -373,6 +307,78 @@ func (m *MongoAggregatePump) HandleWriteErr(err error) error {
 	return err
 }
 
+func (m *MongoAggregatePump) DoAggregatedWriting(ctx context.Context, orgID string, filteredData analytics.AnalyticsRecordAggregate) error {
+	collectionName, collErr := m.GetCollectionName(orgID)
+	if collErr != nil {
+		m.log.Info("No OrgID for AnalyticsRecord, skipping")
+		return nil
```

Comment:
if this returns nil, then the caller function will say that the records were added, which would be weird as they were not. Is an issue to return an error? or some boolean like: `didInsert`